python/felis/tap.py

```diff
@@ -407,7 +407,9 @@ def visit_column(self, column_obj: datamodel.Column, table_obj: Table) -> Tap11B
         felis_type = FelisType.felis_type(felis_datatype.value)
         column.datatype = column_obj.votable_datatype or felis_type.votable_name
 
-        column.arraysize = column_obj.votable_arraysize or column_obj.length
+        column.arraysize = column_obj.votable_arraysize or (
+            column_obj.length if (column_obj.length is not None and column_obj.length > 1) else None
```

Comment:
Is there a case where `column_obj.votable_arraysize` is None `column_obj_length == 1` and we want `column.arraysize `set to that?

Reply:
It says in the TAP 1.1 spec that arraysize should never be set to 1.